dashboard/README.md

```diff
@@ -1,153 +1,86 @@
-## Foreword
-
-  
+# Pbench Dashboard
 
 Pbench Dashboard is a web-based platform for consuming indexed performance benchmark data. The platform provides a consolidated view of benchmark data within tables, charts, and other powerful visualizations. Users are able to quickly navigate through benchmark data and tune analytics through comparison tools for in-depth analysis.
 
-# Dashboard directory structure
+## Dashboard directory structure
 
-## public
+### [public](public/)
 
 Contains the root application `index.html` and React template artifacts.
 
-(BTW, do we really want/need to retain the React logos here? We don't use them, do
-we?)
-
-## server
+### [server](server/)
 
 The source for an NPM express server that's used in local developer mode to reflect
 the server `/api/v1/endpoints` API call to a remote server. This enables local
 debugging against a real Pbench Server.
 
-## src
+### [src](src/)
 
 The Pbench Dashboard Javascript source plus additional CSS/LESS and artifacts.
 
-### assets
+### [src/assets](src/assets/)
 
 Assets placed in the `src/assets/images` directory are only referenced within component or layout definitions and are packaged in the generated `***.js` file during the build process.
 
-### modules
-  
-### reducers
-  
-### store
-
-### utils
-
-## Prerequisites
- 
-
-## Install Node JS
-
-Refer to the [Node.js](https://nodejs.org) official website
+### [src/modules](src/modules/)
 
-  
-
-## Install create-react-app
-
-Install "create-react-app npm package globally. This will help to easily run the project and also build the source files easily. Use the following command to install create-react-app
-
-  
-
-```bash
+`modules` directory has all containers(patent layouts) and components(react components). 
 
-$ npm install -g create-react-app
+### [src/utils](src/utils/)
 
-```
+`utils` directory has all helper/utility scripts.
 
-  
+## Setup
 
-## Cloning and Running the Application Locally 
+1. Install NodeJS - [offical setup guide](https://nodejs.org/en/download/package-manager/)
 
-- Clone the [Pbench code](https://github.com/distributed-system-analysis/pbench) to a local file system
-- Install all the npm packages
+2. Clone the [Pbench repo](https://github.com/distributed-system-analysis/pbench)
 
-Go into the dashboard folder and type the following command to install all npm packages 
+3. Install all the npm packages.
+Navigate to `dashboard` directory and run following command 
+	```bash
+	$ npm install
+	```
 
+	> NOTE:
+Create `.env` file in the root directory(`/dashboard/`) and declare the environment variable `PBENCH_SERVER`.
```

Comment:
s/Create/Create a/

Also, add a space before the open-paren.

Reply:
Done